scipy/signal/tests/test.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import pandas as pd
```

Comment:
I am afraid we cannot use Pandas in our tests. Is there a way you could rewrite it without?

Reply:
> I am afraid we cannot use Pandas in our tests. Is there a way you could rewrite it without?

can you use datetime?

Reply:
Yes of course it's in the standard library. What we cannot do is use libraries we don't have in our testing dependencies.

Reply:
> Yes of course it's in the standard library. What we cannot do is use libraries we don't have in our testing dependencies.

don't even run the test I wrote. I have no idea why I pushed it. I'm working on a new one with the changes you asked for.

Reply:
I think it's ready :D

Reply:
Thank you, this file needs to be removed though.

Reply:
please check the one I wrote in test_signaltools.py. test.py is not correct.